### PR TITLE
#5316 Upgrade scala from 2.13.2 to 2.13.3

### PR DESCRIPTION
**Pull request by chia7712**
### Checklist
- [x] [Review Contribution Guidelines](https://oharastream.github.io/en/docs/master/contributing/)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #5316)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: docker/stream.dockerfile [hunk 1]
```diff
@@ -24,11 +24,12 @@ ARG BRANCH="master"
 ARG COMMIT=$BRANCH
 ARG REPO="https://github.com/oharastream/ohara.git"
 ARG BEFORE_BUILD=""
+ARG SCALA_VERSION=2.13.3
 WORKDIR /testpatch/ohara
 RUN git clone $REPO /testpatch/ohara
 RUN git checkout $COMMIT
 RUN if [[ "$BEFORE_BUILD" != "" ]]; then /bin/bash -c "$BEFORE_BUILD" ; fi
-RUN ./gradlew clean ohara-stream:build -x test -PskipManager
+RUN ./gradlew clean ohara-stream:build -x test -PskipManager -Pscala.version=$SCALA_VERSION
```
Comment (chia7712) [inline docker/stream.dockerfile:32]: a fix to stream dockerfile that the scala version should be configurable.

File: ohara-it/src/test/scala/oharastream/ohara/it/performance/BasicTestPerformance.scala [hunk 2]
```diff
@@ -130,13 +129,14 @@ abstract class BasicTestPerformance extends WithPerformanceRemoteWorkers {
...
       pool.execute(() => {
-        while (!Thread.currentThread().isInterrupted) {
+        var done = false
+        while (!done && !Thread.currentThread().isInterrupted) {
           try {
             input(timeoutOfInputData)
           } catch {
             case interruptedException: InterruptedException =>
               log.error("interrupted exception", interruptedException)
-              break
```
Comment (jackyoh) [inline ohara-it/src/test/scala/oharastream/ohara/it/performance/BasicTestPerformance.scala:139]: scala 2.13.3 不能使用 break 了？

Comment (chia7712) [reply ohara-it/src/test/scala/oharastream/ohara/it/performance/BasicTestPerformance.scala:139]: 不建議

Comment (jackyoh) [reply ohara-it/src/test/scala/oharastream/ohara/it/performance/BasicTestPerformance.scala:139]: 了解，Thank you.

**Pull request merged**